doc/nlspsettings.txt

```diff
@@ -58,11 +58,23 @@ setup({opts})                                           *nlspsettings.setup()*
 
             Default: `"json"`
 
+        {ignored_servers} (optional, table)
+            Add ignored servers, you may not want configure servers such
+            as `null-ls`, `efm-langserver`.
+
```

Comment:
I would like to see a little more explanation added.
Could you add the following explanation? (I am not good at English, so I will leave the text to you.)

List of server names that should not appear in the server choices when running LspSettings buffer and LspSettings local buffer if more than one server is connected to the current buffer

Reply:
Changed